hw/ip/otp_ctrl/dv/cov/otp_ctrl_cov_unr_excl.el

```diff
@@ -174,521 +76,460 @@ ANNOTATION: "VC_COV_UNR"
-Condition 7 "4291765346" "(scrmbl_mtx_gnt_i && scrmbl_ready_i) 1 -1" (1 "01")
-ANNOTATION: "VC_COV_UNR"
-Condition 9 "183597720" "((escalate_en_i != Off) || cnt_err) 1 -1" (2 "01")
-CHECKSUM: "714850099 2365744336"
-INSTANCE: tb.dut.gen_partitions[7].gen_lifecycle.u_part_buf
-ANNOTATION: "VC_COV_UNR"
-Condition 9 "183597720" "((escalate_en_i != Off) || cnt_err) 1 -1" (2 "01")
-ANNOTATION: "VC_COV_UNR"
-Condition 10 "624370688" "((base_sel == DigOffset) ? DigestOffset : 11'b11110101000) 1 -1" (2 "1")
-ANNOTATION: "VC_COV_UNR"
-Condition 11 "705391888" "((data_sel == ScrmblData) ? scrmbl_data_i : otp_rdata_i) 1 -1" (2 "1")
-CHECKSUM: "714850099 1168820513"
-INSTANCE: tb.dut.gen_partitions[4].gen_buffered.u_part_buf
-ANNOTATION: "VC_COV_UNR"
-Condition 2 "1999344180" "(scrmbl_mtx_gnt_i && scrmbl_ready_i) 1 -1" (1 "01")
-ANNOTATION: "VC_COV_UNR"
-Condition 6 "1580211052" "(scrmbl_mtx_gnt_i && scrmbl_ready_i) 1 -1" (1 "01")
-ANNOTATION: "VC_COV_UNR"
-Condition 9 "183597720" "((escalate_en_i != Off) || cnt_err) 1 -1" (2 "01")
-CHECKSUM: "415438371 1676944626"
-INSTANCE: tb.dut.gen_partitions[3].gen_buffered.u_part_buf.u_prim_count
-ANNOTATION: "VC_COV_UNR"
-Condition 3 "2265771294" "(en_i && (gen_cross_cnt_hardening.down_cnt > '0)) 1 -1" (1 "01")
-ANNOTATION: "VC_COV_UNR"
-Condition 3 "2265771294" "(en_i && (gen_cross_cnt_hardening.down_cnt > '0)) 1 -1" (3 "11")
-ANNOTATION: "VC_COV_UNR"
-Condition 4 "2166812438" "((cmp_valid == CmpValid) ? err : ((cmp_valid == CmpInvalid) ? '0 : 1'b1)) 1 -1" (2 "1")
-ANNOTATION: "VC_COV_UNR"
-Condition 5 "2989213225" "((cmp_valid == CmpInvalid) ? '0 : 1'b1) 1 -1" (1 "0")
-CHECKSUM: "415438371 1676944626"
-INSTANCE: tb.dut.gen_partitions[5].gen_buffered.u_part_buf.u_prim_count
-ANNOTATION: "VC_COV_UNR"
-Condition 3 "2265771294" "(en_i && (gen_cross_cnt_hardening.down_cnt > '0)) 1 -1" (1 "01")
+Transition FsmStateError->CheckFailError "7->6"
+Fsm error_q "2210720134"
 ANNOTATION: "VC_COV_UNR"
-Condition 3 "2265771294" "(en_i && (gen_cross_cnt_hardening.down_cnt > '0)) 1 -1" (3 "11")
+Transition FsmStateError->AccessError "7->5"
+Fsm error_q "2210720134"
 ANNOTATION: "VC_COV_UNR"
-Condition 4 "2166812438" "((cmp_valid == CmpValid) ? err : ((cmp_valid == CmpInvalid) ? '0 : 1'b1)) 1 -1" (2 "1")
+Transition CheckFailError->MacroEccCorrError "6->2"
+Fsm error_q "2210720134"
 ANNOTATION: "VC_COV_UNR"
-Condition 5 "2989213225" "((cmp_valid == CmpInvalid) ? '0 : 1'b1) 1 -1" (1 "0")
-CHECKSUM: "415438371 1676944626"
-INSTANCE: tb.dut.gen_partitions[6].gen_buffered.u_part_buf.u_prim_count
+Transition CheckFailError->FsmStateError "6->7"
+Fsm error_q "2210720134"
 ANNOTATION: "VC_COV_UNR"
-Condition 3 "2265771294" "(en_i && (gen_cross_cnt_hardening.down_cnt > '0)) 1 -1" (1 "01")
+Transition CheckFailError->AccessError "6->5"
+Fsm error_q "2210720134"
 ANNOTATION: "VC_COV_UNR"
-Condition 3 "2265771294" "(en_i && (gen_cross_cnt_hardening.down_cnt > '0)) 1 -1" (3 "11")
+Transition AccessError->MacroEccCorrError "5->2"
```

Comment:
This change and below for all unbuffered partition is due to recent OTP code change: the `error_q` does not have `MacroEccUncorrErr`.

Reply:
Yeah I think this is the same justification as above. Once we are in the terminal error state and once `error_q` has been assigned, it cannot be changed to another error code anymore.